GithubApp/Services/API.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  API.swift
+//  GithubApp
+//
+//  Created by BrainX on 07/06/2022.
+//
+
+import Foundation
+import Alamofire
+class API{
+    static let instance = API()
```

Comment:
if you want to make it a singleton class then add a private init method. Also the naming convention for instance of singleton classes is 
`static let shared = API()`